# Define the goal
bool move_allowed
---
# Define the result
string descript
---
# Define a feedback message
string status
